docs/50_eos-evm/30_developer-guide/10_architecture.md

```diff
@@ -8,7 +8,7 @@ The EVM is implemented as a smart contract in the EOS network. Generating transa
 
 To achieve our goal of full RPC compatibility, we utilize a full functioning ethereum node (Geth in current design) to provide all the read APIs while all write access will be forwarded to a small service to pack them into EOS calls to the EVM contract.
```

Comment:
Sentence too long. Split in two:
design) to provide -> design). This allows to provide

will be -> are
ethereum -> Ethereum